cmd/mdmctl/apply_dep_profile.go

```diff
@@ -16,15 +18,23 @@ func (cmd *applyCommand) applyDEPProfile(args []string) error {
 	var (
 		flProfilePath = flagset.String("f", "", "filename of DEP profile to apply")
 		flTemplate    = flagset.Bool("template", false, "print a JSON example of a DEP profile")
+		flAnchorFile  = flagset.String("anchor", "", "filename of PEM certificate to add to trusted anchors")
```

Comment:
Hmm. Well, I haven't tested. I'd think you can just trust the leaf and that's good. I would also think you could just trust the CA (but could see how that wouldn't work, too). I know it works specifying a self-signed, though, but that doesn't address the question.